rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,8 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.15
```

Comment:
so it's not needed now but if it does not do any harm and the template would add it anyway, I think we can keep it?